tests/main/docker-smoke/task.yaml

```diff
@@ -1,7 +1,8 @@
 summary: Check that the the docker snap works basically
 
 # only run on ubuntus for now, the docker snap has issues on non-ubuntu ATM
-# TODO:UC20: enable for UC20
+# TODO:UC20: enable for UC20, fails with what looks like apparmor
+# errors trying to run kmod and open /run/system/sessions/1 and ptrace
```

Comment:
from the log that @mvo5 shared, the issue here is rather with overlayfs mounts, the /run/systemd/sessions/1 and ptrace denials are normal